### PR TITLE
Avro batch format

### PR DESCRIPTION
**Pull request by adyach**
# One-line summary

PR introduces Avro batch format for Nakadi Publishing API.

### CODE REVIEW

File: core-services/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java [hunk 1]
```diff
@@ -28,19 +34,37 @@ public class KafkaRecordDeserializerTest {
...
         schemaService = new TestSchemaProviderService(localSchemaRegistry);
         metadataSchema = localSchemaRegistry.getLatestEventTypeSchemaVersion(LocalSchemaRegistry.METADATA_KEY);
+        nakadiRecordMapper = new NakadiRecordMapper(localSchemaRegistry);
+        deserializer = new KafkaRecordDeserializer(nakadiRecordMapper, singleSchemaProvider, localSchemaRegistry);
     }
 
     @Test
     public void testDeserializeAvro() throws IOException {
-        final KafkaRecordDeserializer deserializer = new KafkaRecordDeserializer(schemaService, localSchemaRegistry);
+        final KafkaRecordDeserializer deserializer = new KafkaRecordDeserializer(
```
Comment (a1exsh) [inline core-services/src/test/java/org/zalando/nakadi/repository/kafka/KafkaRecordDeserializerTest.java:66]: Local variable hides class field with the same name.

**Comment by a1exsh** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Comment by a1exsh** [comment]
:+1:

**Pull request merged**